x/conflict/README.md

```diff
@@ -0,0 +1,124 @@
+
+A group of validators is selected as a jury to determine the fraudulent and honest providers. Through an event, the chain announces the conflict voting period and the participating providers. During the voting period, providers need to submit their hashed response + salt to the original relay request. This is done to prevent other providers from cheating or copying their vote. Once the voting period ends, the conflict moves to the reveal state. In this state, providers need to reveal their response + salt, which is then verified and compared to the original responses. After the reveal period ends, the votes are counted, and the provider with the fewest votes is penalized by having a fraction of their staked tokens taken and distributed among all the participants.
+
+### Commit Period
+
+This is the voting period of the conflict, providers that are selected as jury are required to submit their hashed vote.
+The hash consisnt of nonce + response hash + provider address
+
+```go
+type MsgConflictVoteCommit struct {
+	Creator string // provider address
+	VoteID  string // the conflict unique id
+	Hash    []byte // the hashed nonce + response + provider address
+}
+```
+
+### Reveal Period
+
+Once the commit period ends, the conflict enters reveal state. Here providers need to submit their full response and nonce. this is than validated and compared to the original providers responses. The result of the vote (Provider A, Provider B, None) is than saved on state, waiting for the end of the conflict.
+
+```go
+type MsgConflictVoteReveal struct {
+	Creator string // provider address
+	VoteID  string // the conflict unique id
+	Nonce   int64  // random nonce
+	Hash    []byte // hash of the response
+}
+```
+
+### Conflict Resolve
+
+Once the reveal period ends votes are than counted to determine the results of the conflict. Votes are weighted by providers stake. 
+Providers in the jury that did not vote are punished by jail and slashing, their slashed amount will be added to the conflict reward pool.
+For the conflict resolution there needs to be a majority met of votes for Provider A, Provider B or None of them. 
+If a majority was not met, conflict reward pool is given to the consumer that reported the conflict.
+Once a majority is met providers that voted to the wrong side of the conflict are slashed and frozen, the slashed amount is added to the conflict reward pool.
+Now the reward pool is distributed between the comsumer and the providers that voted for the correct provider.
+
+## Parameters
```

Comment:
when explaining about the parameters below, put another new line between them so in the README each will be in a new line